website/docs/howto.md

```diff
@@ -84,6 +85,49 @@ Note the `match` field, which defines the scope of objects to which a given cons
 
 Note that if multiple matchers are specified, a resource must satisfy each top-level matcher (`kinds`, `namespaces`, etc.) to be in scope. Each top-level matcher has its own semantics for what qualifies as a match. An empty matcher is deemed to be inclusive (matches everything). Also understand `namespaces`, `excludedNamespaces`, and `namespaceSelector` will match on cluster scoped resources which are not namespaced. To avoid this adjust the `scope` to `Namespaced`.
 
+### The parameters field
+
+The `parameters` field describes the intent of a constraint as parameterized and configured by the operator. It can be referenced by the ConstraintTemplate's rego using `input.parameters` as part of the policy evaluation. Gatekeeper populates `input.parameters` with values passed into the `parameters` field in the Constraint.
```

Comment:
Is mentioning the operator necessary? That seems like an implementation detail.

Parameters as defined on the constraint template?

"as part of the policy evaluation -> in the Rego source code"